source/_components/recorder.markdown

```diff
@@ -40,7 +40,7 @@ recorder:
         required: false
         type: int
       purge_interval:
-        description: How often (in days) the purge task runs. If a scheduled purge is missed (e.g., if Home Assistant was not running), the schedule will resume soon after Home Assistant restarts. You can use the [service](#service-purge) call `purge` when required without impacting the purge schedule.
+        description: How often (in days) the purge task runs. If a scheduled purge is missed (e.g., if Home Assistant was not running), the schedule will resume soon after Home Assistant restarts. You can use the [service](#service-purge) call `purge` when required without impacting the purge schedule. If this is set to zero, purging is disabled.
```

Comment:
That field expect an numeric value, wouldn't "If this is set to 0, purging is disabled." be more descriptive?

Reply:
Thanks. I am not sure, so now I changed it to include both options :)